R/reportEmi.R

```diff
@@ -1164,7 +1188,12 @@ reportEmi <- function(gdx, output = NULL, regionSubsetList = NULL,
                setNames(dimSums(vm_emiMacSector[, , "co2luc"], dim = 3) * GtC_2_MtCO2,
                         "Emi|CO2|+|Land-Use Change (Mt CO2/yr)"),
                # negative emissions from (non-BECCS) CDR (DACCS, EW)
-               setNames((vm_emiCdrTeDetail[, , "weathering"] + vm_emiCdrTeDetail[, , "dac"] * p_share_CCS) * GtC_2_MtCO2,
+               setNames((
+                vm_emiCdrTeDetail[, , "weathering"]
+                + vm_emiCdrTeDetail[, , "dac"] * p_share_CCS
+                + (vm_emiCdrTeDetail[, , "oae_ng"] + vm_emiCdrTeDetail[, , "oae_el"])
+                  * (1 - s33_OAE_chem_decomposition * (1 - p_share_CCS))
+               ) * GtC_2_MtCO2,
                         "Emi|CO2|+|non-BECCS CDR (Mt CO2/yr)")
```

Comment:
recalculate to account only for "net CDR". Use emiCDR?

Reply:
If I understand correctly, this variable should be negative emissions, so gross(ish) = gross - released CO2 that is unavoidable due to <100% capture rate. So it should be `vm_emiCdr - vm_emiCdrTeDetail[, , "dac"] * (1 - p_share_CCS)`

E.g.: `Emi|GHG = Emi|CO2|+|Energy (952.393059421011) + Emi|CO2|+|Industrial Processes (18.70015799332) + Emi|CO2|+|Land-Use Change (38.7566666666702) + Emi|CO2|+|non-BECCS CDR (0) + Emi|GHG|+|CH4 (352.029277806816) + Emi|GHG|+|F-Gases (27.202629) + Emi|GHG|+|N2O (76.9251374089826)`